lib/almodovar.rb

```diff
@@ -16,14 +16,27 @@
 require 'almodovar/to_xml'
 
 module Almodovar
-
   class << self
+    DEFAULT_SEND_TIMEOUT = 30
+    DEFAULT_CONNECT_TIMEOUT = 30
+    DEFAULT_RECEIVE_TIMEOUT = 30
+
     def default_options
-      @default_options ||= {
-        :timeout => 120,
-        :connect_timeout => 30,
-        :user_agent => "Almodovar/#{Almodovar::VERSION}"
+      default = {
+        send_timeout: DEFAULT_SEND_TIMEOUT,
```

Comment:
Yeah, it doesn't _really_ matters, but my point is that by trying to keep a symmetry with the library underneath, we're considering less meaningful names.

I'd just go with `timeout` for now, and would add the other ones as soon as someone actually needs them.